app/server/Main.hs

```diff
@@ -1,6 +1,79 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+{-
+   This module parses command line arguments and starts the wallet.
+
+   This should be the only module that has a notion of command line arguments.
+   Although, for development purposes, we may extend the docopt specification
+   here and call `getArgs` where it is needed.
+-}
 module Main where
 
+import Control.Monad
+    ( when )
 import Prelude
+import System.Console.Docopt
+    ( Docopt
+    , docopt
+    , exitWithUsage
+    , getArgOrExitWith
+    , isPresent
+    , longOption
+    , parseArgsOrExit
+    )
+import System.Environment
+    ( getArgs )
+import Text.Read
+    ( readMaybe )
+
+
+-- | Command-Line Interface specification. See http://docopt.org/
+cli :: Docopt
+cli = [docopt|
+cardano-wallet-server
+
+Start the cardano wallet along with its API and underlying node.
+
+Usage:
+  cardano-wallet-server [options]
+  cardano-wallet-server --help
+
+Options:
+  --wallet-server-port <PORT>  port used for serving the wallet API [default: 8090]
+  --node-port <PORT>           port used for node-wallet communication [default: 8080]
+  --network <NETWORK>          mainnet or testnet [default: mainnet]
```

Comment:
Thought about removing `--network` and having  `cardano-wallet-server mainnet` and `cardano-wallet-server testnet`, but that opted for this.

Reply:
What would we do besides `cardano-wallet-server start`? If there's only one command it seems pointless.

Also I have a vague memory of a more extensive CLI mentioned… corresponding to the REST API? Would that be…
1.  a separate executable acting like a wallet client?
2. something like the rust `cardano-cli`, not running in the background
3. something else?

Reply:
Nothing's decided yet. For now, this is really our future small launcher CLI, so yeah, there's be likely a single command. Naming it still makes things explicit though.

Reply:
But you'd have to run `stack exec -- start` instead of `stack exec`. I would rather keep things simple for now. Fine?

Reply:
Hm, or maybe you don't… sure

Reply:
Ho yeah, it's really just a flavor.